tests/integration/device_test.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+include './tests/unit/utils.sh'
+include './tests/integration/utils.sh'
+include './src/device_info.sh'
+
+function oneTimeSetUp()
+{
+  setup_container_environment
+}
+
+# test if `kw device --local in the container matches the host
+function device_info_test_helper()
+{
+  local distro="${1}"
+  local container
+  local buffer
+  local filter
+  local filter_regex
+  local output
+  local expected_output
+
+  # buffer with all information
+  container="kw-${distro}"
+  buffer=$(container_exec "$container" kw device --local)
```

Comment:
Add single-quotes around the kw command.

Reply:
It is not possible to add single-quotes around the kw commands because podman will try to run a file/binary/program called `kw\ device\ --local` instad of running `kw` and passing two arguments: `device` and `--local`.

Therefore, we cannot add single-quotes here.
Also, the way it is now give us more flexibility to run `kw` commands with several flags.